packages/story-editor/src/components/colorPicker/currentColorPicker.js

```diff
@@ -162,65 +160,65 @@ function CurrentColorPicker({ rgb, hsl, hsv, hex, onChange, showOpacity }) {
   });
 
   return (
-    <Container data-testid="colorPicker">
-      <Body showOpacity={showOpacity}>
-        <Suspense fallback={null}>
-          <SaturationWrapper>
-            <Saturation
-              radius="8px"
-              pointer={() => (
-                <Pointer offsetX={-12} offsetY={-12} currentColor={rgb} />
-              )}
-              hsl={hsl}
-              hsv={hsv}
-              onChange={onChange}
-            />
-          </SaturationWrapper>
-          <HueWrapper>
-            <Hue
+    <>
+      <Suspense fallback={null}>
+        <SaturationWrapper>
+          <Saturation
+            radius="8px"
+            pointer={() => (
+              <Pointer offsetX={-8} offsetY={-8} currentColor={rgb} />
```

Comment:
Nit: Worthwhile moving the `8` into a constant?

Reply:
These offsets are slightly magic throughout this file so don't feel like there's a good way to do that.